icons/ubereats.svg

```diff
@@ -1 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Uber Eats icon</title><path d="M0 2.865v4.997c0 1.883 1.332 3.13 3.084 3.13a2.965 2.965 0 0 0 2.15-.877v.743h1.211V2.865H5.223v4.933c0 1.265-.87 2.12-1.995 2.122-1.139-.002-1.997-.834-1.997-2.122V2.865zm7.363 0v7.993h1.163v-.732a2.992 2.992 0 0 0 2.117.876 3.044 3.044 0 1 0 0-6.086c-.79 0-1.55.315-2.107.876V2.865zm9.885 2.056a3.022 3.022 0 0 0-3.035 3.024c0 1.737 1.373 3.037 3.153 3.037a3.123 3.123 0 0 0 2.558-1.243l-.85-.618a2.05 2.05 0 0 1-1.708.858 1.971 1.971 0 0 1-1.97-1.655h4.817v-.379c0-1.734-1.254-3.024-2.963-3.024zm6.163.066a1.594 1.594 0 0 0-1.376.766v-.719h-1.163v5.824h1.175V7.546c0-.902.558-1.484 1.326-1.484h.495V4.989h-.407l-.05-.002zm-6.203.944a1.844 1.844 0 0 1 1.834 1.486h-3.618a1.844 1.844 0 0 1 1.784-1.486zm-6.659.006a2.021 2.021 0 1 1 .002 4.042 2.015 2.015 0 0 1-1.416-.598 2.015 2.015 0 0 1-.585-1.422c0-.554.222-1.056.584-1.422a2.015 2.015 0 0 1 1.415-.6zM0 12.987v7.971h5.722v-1.367H1.546v-1.97H5.61v-1.315H1.546v-1.955h4.176v-1.365zm14.56.41v1.685h-1.15v1.338h1.154v3.143c0 .793.572 1.421 1.6 1.421h1.643l-.006-1.338H16.66c-.348 0-.572-.15-.572-.464v-2.768H17.8v-1.332h-1.706v-1.686zm-5.297 1.527a3.103 3.103 0 1 0 .07 6.205 2.991 2.991 0 0 0 1.912-.666v.532h1.518v-5.913h-1.509v.526a3.005 3.005 0 0 0-1.99-.684zm11.771.007c-1.585 0-2.7.644-2.7 1.886 0 .86.613 1.421 1.936 1.695l1.448.328c.57.11.722.259.722.49 0 .371-.438.603-1.127.603-.876 0-1.378-.19-1.573-.848h-1.533c.22 1.231 1.156 2.05 3.049 2.05h.002c1.752 0 2.742-.819 2.742-1.953 0-.806-.585-1.408-1.809-1.667l-1.294-.26c-.751-.136-.988-.274-.988-.546 0-.357.361-.575 1.03-.575.722 0 1.252.192 1.405.847h1.518c-.086-1.229-.99-2.05-2.827-2.05zm-11.567 1.25c1.01.01 1.819.837 1.807 1.847a1.8 1.8 0 0 1-1.825 1.802 1.824 1.824 0 1 1 .018-3.648z"/></svg>
```

Comment:
I'm seeing some slight differences between PR (red) and source (black), any chance you could take a look?
![image](https://user-images.githubusercontent.com/1830867/103993822-15548a80-518e-11eb-9832-14e9fb3e96f4.png)

Also there's a couple of spots that contain very close anchors. They're passing the linter - but may be worth simplifying the path if at all possible?
![image](https://user-images.githubusercontent.com/1830867/103993892-2f8e6880-518e-11eb-8535-c31e1bdb7abf.png)

Reply:
@adamrusted Should be better now. 💪🏼 But I had to increase the precision to 4.